radis/test/lbl/test_calc.py

```diff
@@ -777,7 +778,7 @@ def test_calc_spectrum_multiple_molecules_wstep_auto(
         wavelength_min=4165,
         wavelength_max=5000,  # cm-1
         isotope="1,2,3",
-        pressure=1.01325,  # bar
+        pressure=10.01325,  # bar
```

Comment:
multiplying the pressure by 10 increases the wstep of each spectrum by 10. So it is effectively something happening in the calculation of the spectra